src/routes/userRouter.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const userController = require('../controllers/userController');
+const router = express.Router();
+
+router.get("/get-tickets/:userId", userController.userTicketsController)
```

Comment:
```suggestion
router.get("/", userController.getTicketInfoByUserId)
```
- RESTful API 규칙을 지키기 위해서는 uri에 동사가 들어가지 않는 방향이 좋습니다.
- userId는 parameter가 아닌 token에서 가져오셔야 하기 때문에 제거하셔야 합니다.